goat

```diff
@@ -151,9 +160,27 @@ link()
 		rm -- "$GOAT_PATH/$1"
 	fi
 
-	ln -fs -- "$targetpath" "$GOAT_PATH/$1"
+	$LN "$targetpath" "$GOAT_PATH/$1"
 }
 
+unix_ln()
+{
+	targetpath="$1"
+	destination="$2"
+	ln -fs -- "$targetpath" "$destination"
+}
+
+mingw_ln()
+{
+	targetpath="$1"
+	destination="$2"
+	#cmd "$destination" "$targetpath"
+	"$GOAT_PATH/mingw_ln.bat" "$destination" "$targetpath"
+exit
+EOF
```

Comment:
```
exit
EOF
```

Why is it here? It is a typo, right?

Reply:
Yes, that is a typo. Oops! I was trying to get the path mangling to work through a here-doc, but it didn't. I'll remove that